src/testUtils/TestRig.react.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+
+export var TestRigComponent = React.createClass({
+  contextTypes: {
+    triggerComplete: React.PropTypes.func.isRequired,
+    finish: React.PropTypes.func
+  },
+  getInitialState: function() {
+    return { keyVal : Math.random()};
```

Comment:
@BJK Unbalanced padding inside the curly braces, and an atypical space before the colon.